home-page.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+<link rel="stylesheet" href="home-page-style.css">
```

Comment:
`-style` can be skipped from the name when the extension assume it's a style file.
```suggestion
<link rel="stylesheet" href="home-page.css">
```
unless you call the entire file simply `styles.css` but in any case this is a nitpick and you don't need to change anything related to it ( :